media/src/androidMain/kotlin/dev/icerock/moko/media/picker/FilePickerFragment.kt

```diff
@@ -25,9 +26,10 @@ class FilePickerFragment : Fragment() {
 
         // TODO нужно убрать использование внешней зависимости, сделать конфигурацию способа
         //  выбора файла из вне (аргументом в контроллер передавать)
-        val intent = Intent(requireContext(), FilePickerActivity::class.java)
-        intent.putExtra(FilePickerActivity.ARG_CLOSEABLE, true)
-        startActivityForResult(intent, requestCode)
+        MaterialFilePicker().withSupportFragment(this)
+            .withCloseMenu(true)
+            .withRequestCode(requestCode)
+            .start()
```

Comment:
also you should up version of used library to 1.9.1 as described in #27 . as i see you use new api

Reply:
set materialfilepicker version to 1.9.1